### PR TITLE
[T5A1][W15-B4]MadasamyRaviNadarMamtha

### PR DESCRIPTION
**Pull request by Mamtha3005**
Add user stories in Development guide

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | give special tags to contacts | categorise people
+`* *` | user | edit contacts | contacts are updated
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: don't you think this deserves a higher priority? :-)

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -57,19 +59,20 @@ Priority | As a ... | I want to ... | So that I can...
...
 
-1. User requests to list persons
-2. AddressBook shows a list of persons
-3. User requests to delete a specific person in the list
-4. AddressBook deletes the person <br>
+1. User requests to edit a person tag
+2. AddressBook display the current person tag
+3. User enters the new person tag
+4. AddressBook asks if change is confirmed.
+5. Addressbook change the tag and store in database.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: user should confirm before the address book changes the tag.

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -57,19 +59,20 @@ Priority | As a ... | I want to ... | So that I can...
 
 **MSS**
 
-1. User requests to list persons
-2. AddressBook shows a list of persons
-3. User requests to delete a specific person in the list
-4. AddressBook deletes the person <br>
+1. User requests to edit a person tag
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: the activity was updated to rename a tag such that all people with the given tag will have the tag updated. please check the description again

File: doc/DeveloperGuide.md [hunk 4]
```diff
@@ -57,19 +59,20 @@ Priority | As a ... | I want to ... | So that I can...
...
+2. AddressBook display the current person tag
+3. User enters the new person tag
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: - what happens when the new tag given is already present in the address book?
- what happens if the new tag is same as the old tag?

File: doc/DeveloperGuide.md [hunk 5]
```diff
@@ -80,7 +83,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
-
+5. Should have a nice GUI.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: how do you define nice GUI, isn't it too subjective?
- in general, try to be as precise and objective as possible.
- words like nice, easy, more, less etc can be avoided and use precise values (where possible)

**Comment by okkhoy** [comment]
@Mamtha3005 
Some comments added. Please ack & close the PR after reading comments.

**Comment by Mamtha3005** [comment]
Thank you so much!

**Pull request closed without merging**